README.md

```diff
@@ -21,7 +21,7 @@ Foundry consists of:
 
 ## Forge Quickstart
 
-Forge is a building and testing framework for Ethereum development.
+Forge is a development and testing framework for Ethereum development.
```

Comment:
i dont like saying development twice tbh

Reply:
Me neither, how's the new version?

Reply:
'for Ethereum' feels a bit ambiguous, like it could be for clients themselves or something. would prefer 'for Ethereum applications' or 'for Ethereum smart contracts'

Reply:
oop looks like it got merged 🤷‍♂️

Reply:
Nows your chance to get another PR in! 😛